module/os/freebsd/spl/spl_uio.c

```diff
@@ -45,6 +45,16 @@
 #include <sys/vnode.h>
 #include <sys/zfs_znode.h>
 
+static void
+zfs_freeuio(struct uio *uio)
+{
+#if __FreeBSD_version > 1500013
+	freeuio(uio);
+#else
+	free(uio_clone, M_IOV);
```

Comment:
```suggestion
	free(uio, M_IOV);
```
?

Reply:
Sigh, thank you. Fixed.

Reply:
And do we need the function if it is used in only one place in the same file?

Reply:
We don't, but I found it a bit nicer this way, and new consumers may be added in the future. If you prefer to have the change inline, I will do that, I don't feel too strongly either way.

Reply:
I generally don't like excessive wrapping, since it obfuscates the code more than cleans up. But I won't object much if you prefer that way.

Reply:
Me neither, but I also don't like ugly `__FreeBSD_version` checks in the middle of functions, so my slight preference is to keep the wrapper.